go/integTest/shared_commands_test.go

```diff
@@ -3478,6 +3478,39 @@ func (suite *GlideTestSuite) TestPExpireTime() {
 	})
 }
 
+func (suite *GlideTestSuite) Test_ZCard() {
+	suite.runWithDefaultClients(func(client api.BaseClient) {
+		key := "{key}" + uuid.NewString()
+		membersScores := map[string]float64{
+			"one":   1.0,
+			"two":   2.0,
+			"three": 3.0,
+		}
+
+		res1, err := client.ZAdd(key, membersScores)
+		assert.Nil(suite.T(), err)
+		assert.Equal(suite.T(), int64(3), res1.Value())
+
+		res2, err := client.ZCard(key)
+		assert.Nil(suite.T(), err)
+		assert.Equal(suite.T(), int64(3), res2.Value())
+
+		/**
+		  TODO:
+		  expect(await client.zrem(key, ["one"])).toEqual(1);
+		  expect(await client.zcard(key)).toEqual(2);
+		*/
+
+		//         res3, err := client.ZRem(key, ["one"])
```

Comment:
Use custom command and add a TODO comment.

Reply:
Or rebase on #2880